rocAL/rocAL/include/meta_data/coco_meta_data_reader.h

```diff
@@ -44,7 +44,7 @@ class COCOMetaDataReader: public MetaDataReader
     pMetaDataBatch _output;
     std::string _path;
     int meta_data_reader_type;
-    void add(std::string image_name, BoundingBoxCords bbox, Labels labels, ImgSize image_size, uint image_id = 0);
+    void add(std::string image_name, BoundingBoxCords bbox, Labels labels, ImgSize image_size, int image_id = 0);
```

Comment:
Why image_id has been changed from uint to int?

Reply:
It is supposed to be in int only, In our previous PR it was given uint so it is fixed here